pkg/controller/statefulset/stateful_set.go

```diff
@@ -52,6 +52,10 @@ var controllerKind = apps.SchemeGroupVersion.WithKind("StatefulSet")
 // podKind contains the schema.GroupVersionKind for pods.
 var podKind = v1.SchemeGroupVersion.WithKind("Pod")
 
+const StatefulSetIndex = "statefulsetName"
```

Comment:
`statefulSetName` seems more popular ( upper case `S` for `Set`). Does this string matter at all?

https://cs.k8s.io/?q=statefulSet&i=nope&literal=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

Reply:
It's just used as a key in Indexer map. I don't have strong opinion on naming for this.